winml/lib/Api/LearningModelSession.cpp

```diff
@@ -255,13 +221,31 @@ LearningModelSession::Run(
   auto device = device_.as<LearningModelDevice>();
   CWinMLAutoLock lock(!device->IsCpuDevice() ? &evaluate_lock_ : nullptr);
   // TODO : set the run_options
-  onnxruntime::RunOptions run_options;
+  Ort::RunOptions run_options;
+  binding_impl->BindUnboundOutputs();
```

Comment:
>BindUnboundOutputs [](start = 16, length = 18)

where did this used to happen? was it always part of run or is this a behavior change?

Reply:
This used to happen in "GetIOBinding" and it was just a static function that just took a binding_impl and model and bound unbound outputs. I figured the implementation of this code should live in LearningModelBinding.